recipe/build_pytorch.sh

```diff
@@ -120,6 +138,7 @@ if [[ ${cuda_compiler_version} != "None" ]]; then
     export USE_STATIC_CUDNN=0
     export CUDA_TOOLKIT_ROOT_DIR=$CUDA_HOME
     export MAGMA_HOME="${PREFIX}"
+    export USE_MKLDNN=1
```

Comment:
@ngam can we revert this? i feel like it is giving problems.

Reply:
i don't think it is related to this. but i can't figure out what at the moment. I'm trying to build a single version so I can share logs again.